docs/upgrade/2.5.2_to_2.6.0.rst

```diff
@@ -0,0 +1,132 @@
+.. _latest_upgrade_guide:
+
+Upgrade from 2.5.2 to 2.6.0
+===========================
+
+Update Servers to SecureDrop 2.6.0
+----------------------------------
+Servers running Ubuntu 20.04 will be updated to the latest version of SecureDrop
+automatically within 24 hours of the release.
+
+Update Workstations to SecureDrop 2.6.0
```

Comment:
Should we add a reminder/pointer about the Tails advisory?

Reply:
I think it's a good idea! Just pushed a new commit with that added.